client/cli/src/params/import_params.rs

```diff
@@ -73,14 +73,20 @@ pub struct ImportParams {
 	pub execution_strategies: ExecutionStrategiesParams,
 
 	/// Specify the state cache size.
+	///
+	/// Providing `0` will disable the cache.
 	#[clap(long, value_name = "Bytes", default_value = "67108864")]
-	pub state_cache_size: usize,
+	pub trie_cache_size: usize,
```

Comment:
Existing CLI option should be deprecated rather than removed. We don't want to break existing setups without a warning.